examples/openshift/workshop/watch.json

```diff
@@ -14,7 +14,14 @@
         "name": "CCP_IMAGE_TAG",
         "description": "the image tag to use"
     }, {
+<<<<<<< HEAD
```

Comment:
@xenophenes there is a conflict here which needs to be resolved first

Reply:
Thanks! Resolved.